boards/xtensa/m5stickc_plus/m5stickc_plus.dts

```diff
@@ -80,16 +80,65 @@
 	clock-frequency = <I2C_BITRATE_STANDARD>;
 	sda-gpios = <&gpio0 21 GPIO_OPEN_DRAIN>;
 	scl-gpios = <&gpio0 22 GPIO_OPEN_DRAIN>;
-	pinctrl-0 = <&i2c0_sda_gpio21 &i2c0_scl_gpio22>;
+	pinctrl-0 = <&i2c0_default>;
 	pinctrl-names = "default";
+
+	pmic_xpowers_axp192: pmic@34 {
+		compatible = "xpowers,axp192";
+		reg = <0x34>;
+
+		pmic_axp192_regulators: regulators {
+			compatible = "xpowers,axp192-regulator";
+
+			pmic_axp192_buck1: BUCK1 {
+				/* 700mV~3500mV@25mV/step, max output 1200mA */
+				regulator-init-microvolt = <3300000>;
+				regulator-boot-on;
+			};
+
+			pmic_axp192_buck2: BUCK2 {
+				/* 700mV~2275mV@25mV/step, max output 1600mA */
+				regulator-init-microvolt = <2275000>;
+				regulator-boot-on;
+			};
+
+			pmic_axp192_buck3: BUCK3 {
+				/* 700mV~3500mV@25mV/step, max output 700mA */
+				regulator-init-microvolt = <3300000>;
+				regulator-boot-on;
+			};
+
+			pmic_axp192_ldo1: LDO1 {
+				/* Always on */
+				regulator-always-on;
+			};
+
+			pmic_axp192_ldo2: LDO2 {
+				/* 1800mV~3300mV@100mV/step, max output 200mA */
+				regulator-init-microvolt = <3300000>;
+				regulator-boot-on;
+			};
+
+			pmic_axp192_ldo3: LDO3 {
+				/* 1800mV~3300mV@100mV/step, max output 200mA */
+				regulator-init-microvolt = <3300000>;
+				regulator-boot-on;
+			};
+
+			pmic_axp192_ldoio0: LDOIO0 {
+				/* 1800mV~3300mV@100mV/step, max output 50mA */
+				regulator-init-microvolt = <3300000>;
+				regulator-boot-on;
+			};
+		};
+	};
```

Comment:
Wouldn't it make sense to have here only the settings that are generic (if any? if not, just declare the regulator, with no other setting) and then "turn on" the TFT backlight and TFT controller in the overlay of the LVGL sample?
I think it's important that people understand at first glance that there is an easy way to completely cutoff all LCD-related components by using the regulator. It also helps have the "default" devicetree get closer to demonstrate minimal power consumption (although my understanding is that power-management is not well/fully supported yet for the ESP-32 port of Zephyr).

Reply:
You should actually make use of `xxx-supply = <&regulator>;` in the display, and be the display driver who controls the regulator on/off state.

Reply:
Sorry, not sure how to write this